CMakeLists.txt

```diff
@@ -110,22 +110,21 @@ include(CMakePackageConfigHelpers)
 
 # write QSkinnyConfigVersion.cmake file for cmake import
 write_basic_package_version_file(
-    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}ConfigVersion.cmake
+    ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}ConfigVersion.cmake
     VERSION ${PACKAGE_VERSION}
     COMPATIBILITY AnyNewerVersion)
 
 # Copy QSkinnyConfig.cmake to build dir
 configure_file(${QSK_CMAKE_DIR}/${PACKAGE_NAME}Config.cmake
-    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}Config.cmake
+    ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}Config.cmake
```

Comment:
Subjectively I don't think we need an additional sub directory in the `CMAKE_CURRENT_BINARY_DIR` since it is already QSkinny's own build folder. It's not wrong, just not necessary... 

> Using `CMAKE_CURRENT_BINARY_DIR` makes us support the "QSkinny as a sub folder" scenario